buildpacks/README.md

```diff
@@ -0,0 +1,44 @@
+# Buildpack V3
+
+This build template builds source into a container image using [Cloud Native Buildpacks](https://buildpacks.io).
+
+The Cloud Native Buildpacks website describes v3 buildpacks as:
+
+> ... pluggable, modular tools that translate source code into container-ready artifacts
+> such as OCI images. They replace Dockerfiles in the app development lifecycle with a higher level
+> of abstraction. ...  Cloud Native Buildpacks embrace modern container standards, such as the OCI
+> image format. They take advantage of the latest capabilities of these standards, such as remote
+> image layer rebasing on Docker API v2 registries.
+
+**Note**: The current Cloud Foundry buildpacks are available in the [CF template](README-CF.md).
+
+## Create the template
+
+```
+kubectl apply -f https://raw.githubusercontent.com/knative/build-templates/master/buildpacks/cnb.yaml
+```
+
+## Parameters
+
+* **IMAGE:** The Docker image name to apply to the newly built image. (_required_)
+* **RUN_IMAGE:** The run image buildpacks will use as the base for IMAGE. (_default:_ `packs/run`)
+* **USE_CRED_HELPERS:** Use Docker credential helpers. Set to `"true"` or `"false"` as string values. (_default:_ `"true"`)
+
+## Usage
+
+```
+apiVersion: build.knative.dev/v1alpha1
+kind: Build
+metadata:
+  name: buildpackv3-example-build
+spec:
+  source:
+    git:
+      url: https://github.com/my-user/my-repo
+      revision: master
+  template:
+    name: cnb
```

Comment:
`cnb` is not very descriptive, how about just `buildpack` ?

Reply:
Maybe `buildpack-cnb` for this, and `buildpack-cf` for the CF one?

Reply:
I think the acronyms `cnb` and `cf` might be confusing for people who are not already experts in the area.

I'd like to get to the point (maybe not in the PR, but soonish) where we speak in terms of `buildpack` and `buildpack-old` (or `buildpack-cloudfoundry` or something else), so users who come here saying "I'd like to use buildpacks" know which one is for them.

Reply:
How about `buildpack-cnb` and `buildpack-cloudfoundry` -- just for now?

These templates are targeted at different users:
- The Cloud Foundry template provides production-ready CF buildpacks for app devs
- The CNB template is for buildpack creators who want to create/run CNB-compatible buildpacks on knative.

I'd like to get to a point where both templates are CNB templates, but one provides Cloud Foundry buildpacks and the other provides riff function invoker buildpacks. (@jldec)